lua/terra-core/types.lua

```diff
@@ -97,7 +97,7 @@
 
 -------------------------------------------Config
 ---@alias TerraConfig.Variant                    "day" | "night"
----@alias TerraConfig.Theme                      "spring" | "summer" | "fall" | "winter" | "control_beta" | "kurzgesagt_beta"
```

Comment:
We temporarily remove the beta flag until we introduce a more sophisticated Theme definition map. The new map will include a field for handling the development status.